src/ontology/cl-edit.owl

```diff
@@ -30124,7 +30124,7 @@ SubClassOf(obo:CL_4023188 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0013607
 
 # Class: obo:CL_4023189 (parasol ganglion cell of retina)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32533915") Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Parasol_cell&oldid=1032171532") obo:IAO_0000115 obo:CL_4023189 "A retinal ganglion cell ocated in the ganglion cell layer of the retina and projects to magnocellular cells in the lateral geniculate nucleus (LGN) as part of the magnocellular pathway in the visual system. They have large cell bodies as well as extensive branching dendrite networks and as such have large receptive fields.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32533915") Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Parasol_cell&oldid=1032171532") obo:IAO_0000115 obo:CL_4023189 "A retinal ganglion cell located in the ganglion cell layer of the retina. This cell projects to magnocellular cells in the lateral geniculate nucleus (LGN), as part of the magnocellular pathway in the visual system. They have large cell bodies, as well as extensive branching dendrite networks, and as such, have large receptive fields.")
```

Comment:
Consider:

- "... This cell projects to magnocellular cells in the lateral geniculate nucleus (LGN) as part of the magnocellular pathway in the visual system. They have large cell bodies and extensive, branching dendritic networks that contribute to their large receptive fields."

- "as part of the magnocellular pathway in the visual system" seems redundant.